### DISCUSSION TITLE
Is there any other way could I know if Kyuubi event custom handler be configured correct?

### DISCUSSION THREAD

**Discussion by madeirak** [apache/kyuubi#6791] [Q&A]
After configuring Kyuubi custom event handler， kyuubi=1.7.1（https://kyuubi.readthedocs.io/en/v1.7.1-rc0/extensions/server/events.html）, There were no expected results during the self-test process. Is there any other way could I know if custom handler be configured correct?

/custom_event_handler/src/main/scala/custom_event_handler_case1/CustomEventHandlerCase1.scala
```
package org.apache.kyuubi.events

import org.apache.kyuubi.events.handler.{CustomEventHandlerProvider, EventHandler}
import org.apache.kyuubi.config.KyuubiConf
import scalaj.http._


class CustomEventHandlerCase1Provider extends CustomEventHandlerProvider {
  override def create(kyuubiConf: KyuubiConf): EventHandler[KyuubiEvent] = {
    new CustomEventHandlerCase1(kyuubiConf)
  }
}

class CustomEventHandlerCase1(kyuubiConf: KyuubiConf) extends EventHandler[KyuubiEvent] {

  override def apply(kyuubiEvent: KyuubiEvent): Unit = {
    //real logic
    println("+++++++++++++++++++++++++"+kyuubiEvent)
  }
}
```

/custom_event_handler/src/main/resources/META_INF/services/org.apache.kyuubi.events.handler.CustomEventHandlerProvider 
```
custom_event_handler_case1.CustomEventHandlerCase1Provider
```

Added configuration `kyuubi.backend.server.event.loggers=CUSTOM`

**Comment by pan3793** [comment]
@madeirak thanks for opening the discussion, the question might not be answered timely due to the lack of community volunteer resources, I think you have found the solution so closed it, would you mind leaving a summary before closing? It might help other guys who have similar questions in the future, thank you in advance

**Reply by madeirak** [reply]
Unfortunately, I used another method..
